dashboards/migrations/0005_auto_20170308_0831.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-08 08:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboards', '0004_definition'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='definition',
+            options={},
+        ),
+    ]
```

Comment:
No idea why it that was removed, but I would prefer we keep up to date with migrations, so let's just leave it in this PR, it's easier than creating another PR for it.